packages/blade/src/components/Tabs/__tests__/Tabs.native.test.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react-native';
+import { Tabs, TabList, TabItem, TabPanel } from '..';
+import renderWithTheme from '~utils/testing/renderWithTheme.native';
+import { Text } from '~components/Typography';
+
```

Comment:
no tests for testing href, testing the controlled state?

Reply:
Testability issues: https://razorpay.slack.com/archives/G01B3LQ9H0W/p1698213664463119